tasks/steps/blocks/update_package.yml

```diff
@@ -6,9 +6,10 @@
     package_path: '{{ delivered_package_path }}'
     app_name: '{{ cartridge_app_name }}'
 
-- name: 'Set "cartridge_app_name" from package info'
+- name: Set cartridge_app_name from package info
   set_fact:
     cartridge_app_name: '{{ package_info.name }}'
+  when: package_info.type in ['rpm', 'deb']
```

Comment:
Let's do `cartridge_app_name` not required or remove this task

Reply:
Removed this task and computing package name in `set_package_info`.